solutions/devsprint-eduardo-bocato-1/SpaceApp/Modules/Features/Home/Scenes/Home/HomeScene.swift

```diff
@@ -3,31 +3,53 @@ import SwiftUI
 
 struct HomeScene: View {
     @StateObject var viewModel: HomeViewModel
-    
+    @State private var tabSelected: Int = 1
```

Comment:
**[CHANGE REQUEST]**
Please use the viewModel to handle this, otherwise it won't be able to control or act on it.

```swift
// on HomeState
struct HomeState: Equatable {
    // ...
    var selectedTabIndex: Int
}

// Then, on the view...
struct HomeView: View {
    // ...
    var body: some View {
         TabView(selection: $viewMoldel.selectedTab) { // to do this, you can remove the private(set) from the state
}

Reply:
Yeap sounds better! As you explained all variables and bindings needs be on state, in our case `HomeState`, I'll update my branch with these suggestions, thanks